platform/viewer/cypress/integration/common/OHIFDownloadSnapshotFile.spec.js

```diff
@@ -0,0 +1,152 @@
+describe('OHIF Download Snapshot File', () => {
+  before(() => {
+    cy.openStudy('MISTER^MR');
+    cy.waitDicomImage();
+    cy.expectMinimumThumbnails(5);
+  });
+
+  beforeEach(() => {
+    cy.openDownloadImageModal();
+  });
+
+  afterEach(() => {
+    cy.get('[data-cy="close-button"]')
+      .scrollIntoView()
+      .click();
+  });
+
+  it('checks displayed information for Tablet experience', function() {
+    // Set Tablet resolution
+    cy.viewport(1000, 660);
+    // Visual comparison
+    cy.screenshot('Download Image Modal - Tablet experience');
+    cy.percyCanvasSnapshot('Download Image Modal - Tablet experience');
+  });
+
+  it('checks displayed information for Desktop experience', function() {
+    // Set Desktop resolution
+    cy.viewport(1750, 720);
+    // Visual comparison
+    cy.screenshot('Download Image Modal - Desktop experience');
+    cy.percyCanvasSnapshot('Download Image Modal - Desktop experience');
+    //Check if all elements are displayed
+    cy.get('[data-cy=OHIFModal-header]')
+      .as('downloadImageModal')
+      .should('contain.text', 'Download High Quality Image');
+    // Check input fields
+    cy.get('[data-cy=file-info-container]')
+      .should('contain.text', 'Image width (px)')
+      .should('contain.text', 'Image height (px)')
+      .and('contain.text', 'File name')
+      .and('contain.text', 'File type')
+      .and('contain.text', 'Show Annotations');
+    cy.get('[data-cy=file-type]')
+      .select('png')
+      .should('have.value', 'png')
+      .select('jpg')
+      .should('have.value', 'jpg');
+    // Check image preview
+    cy.get('[data-cy="image-preview"]').should('contain.text', 'Image Preview');
+    cy.get('[data-cy="viewport-preview-img"]')
+      .should('have.attr', 'src')
+      .and('include', 'data:image');
+    // Check buttons
+    cy.get('[data-cy="cancel-btn"]')
+      .scrollIntoView()
+      .should('have.text', 'Cancel');
+    cy.get('[data-cy="download-btn"]')
+      .scrollIntoView()
+      .should('have.text', 'Download');
```

Comment:
If we can't test the actual download, verifying the button exists would be the next best thing

Reply:
We can only if we use the workaround:
```
       if (window.Cypress) {
        console.log(window.downloadUrl);
        return;
      }
```

Reply:
I would prefer we not use that workaround for now